src/components/common/ExpandableList.tsx

```diff
@@ -1,92 +1,176 @@
+              <ShadowedView style={[styles.shadowViewStyle, Platform.OS === 'android' && styles.shadowViewStyleAndroidAdjust]}>
+                <ScrollView keyboardShouldPersistTaps="always" style={styles.scrollContainer} scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}>
+                  {items}
+                </ScrollView>
+                {!isExpanded ? null : (
+                  <Animated.View style={[styles.fadeoutContainer, aFadeoutStyle]}>
+                    <GradientFadeOut />
+                  </Animated.View>
+                )}
+              </ShadowedView>
+            </Animated.View>
+          )
+        })
+      } else {
+        localBridge?.resolve(undefined)
+      }
+
+      // Cleanup
+      return () => {
+        localBridge?.resolve(undefined)
+      }
+    },
+    [sAnimationMult, isExpanded, items, dropdownLayoutStyle, itemHeight],
+    'ExpandableList'
+  )
+
+  // Measure the anchor view position and width
+  React.useLayoutEffect(() => {
+    if (containerLayout == null && anchorViewRef.current != null) {
+      anchorViewRef.current.measureInWindow((x, y, width, height) => {
+        // iOS and Android return various garbage values initially.
+        // We expect positive nonzero values. We can always safely omit 0 values
+        // because we guarantee some amount of margin under correct UI design
+        if (x <= 0 || y <= 0 || width <= 0) return
 
-  React.useEffect(() => {
-    setIsAnimateItemsNumChange(isExpanded)
-  }, [items, isExpanded])
+        setContainerLayout({ x, y: Platform.OS === 'android' ? y + 25 : y, width: widthRem == null ? width : theme.rem(widthRem) })
+      })
+    }
+  })
+
+  const handleRowLayout = (event: { nativeEvent: { layout: { height: number } } }) => {
+    if (event != null && itemHeight === 0) {
+      const { height } = event.nativeEvent.layout
+      setItemHeight(height)
+    }
+  }
 
   return (
-    <View style={styles.relativeContainer}>
-      <Animated.View style={[styles.dropdownContainer, widthStyle, aContainerStyle]}>
-        <ScrollView keyboardShouldPersistTaps="always" nestedScrollEnabled scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}>
-          {items}
-        </ScrollView>
-        <GradientFadeOut />
-      </Animated.View>
+    <View ref={anchorViewRef} style={styles.anchorContainer} collapsable={false}>
+      {items.length === 0 ? null : (
+        <View style={styles.dummyMeasureContainer} onLayout={handleRowLayout}>
+          {items[0]}
+        </View>
+      )}
     </View>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  relativeContainer: {
-    position: 'relative',
-    zIndex: 1
+  anchorContainer: {
+    marginHorizontal: theme.rem(0.5)
   },
   dropdownContainer: {
     position: 'absolute',
-    top: '100%',
-    backgroundColor: theme.modal,
+    borderRadius: theme.rem(0.5)
+  },
+  dummyMeasureContainer: {
+    position: 'absolute',
+    opacity: 0
+  },
```

Comment:
I think we want `pointerEvents="none"` on the view, so taps can pass through. Even though it's invisible, it might still affect the touch hierarchy.